compliance-report/generate.sh

```diff
@@ -2,6 +2,23 @@
 
 set -eux
 
+# Get release creator.
+cd $GITHUB_WORKSPACE
+export GH_TOKEN=${GH_TOKEN:-TOKEN}
+RELEASE_CREATOR=$(gh api users/${GITHUB_ACTOR} --jq '.name')
+
+# Handle security report.
+SECURITY_REPORT="N/A"
+if [ -n "$SECURITY_REPORT_LOCATION" ]; then
+  if [[ $SECURITY_REPORT_LOCATION == https* ]]; then
+    SECURITY_REPORT="See $SECURITY_REPORT_LOCATION"
+  else
+    SECURITY_REPORT="See https://github.com/$GITHUB_REPOSITORY/blob/$RELEASE_VERSION/$SECURITY_REPORT_LOCATION"
+  fi
+elif [ -n "$SECURITY_REPORT_URL" ]; then
+    SECURITY_REPORT="See $SECURITY_REPORT_URL"
+fi
+
 cat << EOF >> ${S3_ASSETS}/ssdlc_compliance_report.md
```

Comment:
One more thing I noticed: since we're creating a markdown file here, the individual line breaks get removed, breaking the formatting of the file. Should we just make this a text file instead, like we do with the authorized publication document?

Reply:
Done